pkg/mermaid/integrationdiagram/integrationdiagram.go

```diff
@@ -15,12 +15,30 @@ type integrationPair struct {
 }
 
 //GenerateIntegrationDiagram accepts an application name as input and returns a string (and an error if any)
-//The resulting string is the mermaid code for the integration diagram for that application
+//The resulting string is the mermaid code for the integration diagram
 func GenerateIntegrationDiagram(m *sysl.Module, appName string) (string, error) {
+	if appName == "" {
+		return generateEntireIntegrationDiagramHelper(m, &[]integrationPair{})
+	}
 	return generateIntegrationDiagramHelper(m, appName, &[]integrationPair{}, true)
 }
 
-//generateIntegrationDiagramHelper is a helper which has additional arguments which need not be entered by the user
+//generateEntireIntegrationDiagramHelper is a helper which is generates an entire integration diagram
```

Comment:
I think the function name is a bit confusing, at least I can't seem to fully grasp what it does from the name.

Does it general an integration diagram for ALL applications?
Maybe `generateFullDiagram` or `generateAll`

Reply:
Yep, I'll change that!